metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_query.py

```diff
@@ -120,6 +122,9 @@ def get_all_tags_on_object_with_propagation(
 
     @staticmethod
     def get_all_tags_in_database_without_propagation(db_name: str) -> str:
+
+        allowed_object_domains = f"('{SnowflakeObjectDomain.DATABASE}', '{SnowflakeObjectDomain.SCHEMA}', '{SnowflakeObjectDomain.TABLE}', '{SnowflakeObjectDomain.COLUMN}')"
```

Comment:
Can you make sure that this produces the appropriate string? I thought when I tested this out myself on some bigquery changes, the `str` of a `StrEnum` value like `SnowflakeObjectDomain` didn't produce what I wanted:

```
In [1]: from datahub.ingestion.source.snowflake.constants import SnowflakeObjectDomain

In [2]: str(SnowflakeObjectDomain.DATABASE)
Out[2]: 'SnowflakeObjectDomain.DATABASE'
```

Reply:
Using these in formatted string returns correct results

```
>>> from datahub.ingestion.source.snowflake.constants import SnowflakeObjectDomain
>>> f"{SnowflakeObjectDomain.DATABASE}"
'database'

```

Reply:
Wow, today I learned that f strings call __format__ and not __str__. Very surprised that they're different... if I want to format a value to a string is the most effective way really `f"{value}"`?

Reply:
looks like that.